components/brave_rewards/resources/page/components/pageWallet.tsx

```diff
@@ -107,16 +124,16 @@ class PageWallet extends React.Component<Props, State> {
     if (document.location) {
       const win = window.open(document.location.href)
       if (win) {
-        win.document.body.innerText = utils.constructBackupString(backupKey) // this should be text, not HTML
+        win.document.body.innerText = constructBackupString(backupKey) // this should be text, not HTML
```

Comment:
Should `utils.constructBackupString` go away or is it still used?

Reply:
Removed.